### ISSUE TITLE
Add font_tiny5

### ISSUE THREAD

**Issue by Gissio** [google/fonts#7149]
**Font Project Git Repo URL:**
https://github.com/Gissio/font_tiny5

**Super short description of the Font Family:**
font_tiny5 is a variable-width, 5-pixel font playing with the concept of least amount of information that produces both legible and aesthetically pleasing text.

**Requirements:**

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [x] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I am the author of the font, or the author agreed to this submission
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**
![Font preview](https://github.com/Gissio/font_tiny5/blob/main/documentation/sample.png?raw=true)

Attach here a pic or a screenshot of the font. One image is enough, it can be a few letters, to give a quick overview.

**Comment by 01582** [comment]
nice font

**Comment by 01582** [comment]
i would like to say please read the guide for adding a new font

**Comment by Gissio** [comment]
Thanks for your comments. Could you please point me to what is wrong?

**Comment by 01582** [comment]
> Thanks for your comments. Could you please point me to what is wrong?

You should try on adobe fonts, or try to make pull request by yourself

**Comment by 01582** [comment]
okay since you' done very well but i have an doubts?
did you make your [agrements](https://cla.developers.google.com/) on 
https://cla.developers.google.com

else wait

**Comment by davelab6** [comment]
@Gissio thanks for submitting this, I think it looks great! And I am impressed you extended the glyphs to our African set :) 

BTW @01582 The CLA is not needed if you are not making a PR by yourself, the people doing onboarding can make the PRs and they have signed (and have contracts ;) with Google to get things done :) 

Since this has SFD sources as the 'preferred form of modification' and the UFO exported for convenience, I am assigning to @simoncozens since he's working to improve babelfont and support building more heterogeneous sources, gf library wide. 

Since it is a single style, that might make building from SFD easier too :) 

Also, I foresee no additional family styles, I think the name doesn't need `One` appended.

**Comment by simoncozens** [comment]
Thanks, Dave, I am going to have fun with this one!

@Gissio, this is very cool, but to meet our naming requirements I may have to rename this family to `Tiny5` - but I can do that in production. There are a few other things we will need to change to meet other requirements; I'll send you a PR.

**Comment by simoncozens** [comment]
Sure, that would be great. We're not quite ready to start onboarding, so more changes now are fine.

**Comment by Gissio** [comment]
Excellent! I'll let you know once I finished updating the repository.

**Comment by Gissio** [comment]
Simon:

I've updated the .sfd, .kbitx and .bdf files with the Greek Core, Cyrillic Core and Cyrillic Plus glyph sets. I've also added uni1d7b as "dotless small i with stroke".

I've also noticed an error in the "Build font" GitHub action. The error is:

    FAILED: /tmp/tmp5wp15rmq 
    /home/runner/work/font_tiny5/font_tiny5/venv/bin/python3 -m gftools.builder.jobrunner fontmake --output-path /tmp/tmp5wp15rmq -o ttf -u tiny5-Medium.ufo --filter '...' --filter FlattenComponentsFilter --filter DecomposeTransformedComponentsFilter 

    Command failed:
    fontmake --output-path /tmp/tmp5wp15rmq -o ttf -u tiny5-Medium.ufo --filter ... --filter FlattenComponentsFilter --filter DecomposeTransformedComponentsFilter

    INFO:fontmake.font_project:Building TTF for tiny5-Regular
    INFO:ufo2ft:Pre-processing glyphs
    INFO:ufo2ft.filters:Running DecomposeTransformedComponentsFilter on tiny5-Regular
    INFO:ufo2ft.filters:Running DecomposeComponentsFilter on tiny5-Regular
    INFO:ufo2ft.filters:Running RemoveOverlapsFilter on tiny5-Regular
    ERROR:ufo2ft.filters.removeOverlaps:Failed to remove overlaps for afii10048
    fontmake: Error: In 'tiny5-Medium.ufo': Compiling UFO failed: ('Trying to perform operation on unsupported segment type.', 'qcurve')

    ninja: build stopped: subcommand failed.
    make: *** [Makefile:28: build.stamp] Error 1
    Error: Process completed with exit code 2.

**Comment by simoncozens** [comment]
Nice, thank you. I will take another pass over the African language support.

**Comment by Gissio** [comment]
I merged the PR.

**Comment by Gissio** [comment]
Hi,

I just noticed https://github.com/google/fonts/issues/7596. If you need a short designer profile, I uploaded it to [https://github.com/Gissio/font_tiny5](https://github.com/Gissio/font_tiny5).

**Comment by vv-monsalve** [comment]
Hi @Gissio, please provide the information by filling out [this form](https://docs.google.com/forms/d/1HinKkdCPbHTaiXxakeaUpDo1qsW3P9ZFemK65yWkru0/edit).

**Comment by Gissio** [comment]
Done.

**Comment by Gissio** [comment]
Is there anything else I need to do?

**Comment by simoncozens** [comment]
Nope, we'll test and move through the server pipeline until it goes live...
